pkg/collector/corechecks/networkconfigmanagement/networkconfigmanagement.go

```diff
@@ -186,3 +187,16 @@ func (c *Check) FindMatchingProfile() (*profile.NCMProfile, error) {
 	}
 	return nil, fmt.Errorf("unable to find matching profile for device %s", c.checkContext.Device.IPAddress)
 }
+
+func (c *Check) getDeviceTags() []string {
+	// TODO: confirm what tags should be associated for a device config and/or its metrics
```

Comment:
maybe the config source as well ? in this case it would be hard coded but can help differentiate between the different nc checks in the future ?